$OpenBSD$

/usr/bin/ld: error: duplicate symbol: qh_fprintf
>>> defined at userprintf_r.c
>>>            userprintf_r.c.o:(qh_fprintf) in archive xs/src/qhull/libqhull.a
>>> defined at QhullQh.cpp
>>>            QhullQh.cpp.o:(.text+0x7C0) in archive xs/src/qhull/libqhull.a

See xs/src/qhull/src/libqhullcpp/QhullQh.cpp

Index: xs/src/qhull/src/libqhull_r/userprintf_r.c
--- xs/src/qhull/src/libqhull_r/userprintf_r.c.orig
+++ xs/src/qhull/src/libqhull_r/userprintf_r.c
@@ -1,3 +1,4 @@
+#if 0
 /*<html><pre>  -<a                             href="qh-user_r.htm"
   >-------------------------------</a><a name="TOP">-</a>
 
@@ -63,3 +64,4 @@ void qh_fprintf(qhT *qh, FILE *fp, int msgcode, const 
 
 } /* qh_fprintf */
 
+#endif /* 0 */
